custom/CustomCAN.java

```diff
@@ -62,7 +63,9 @@ protected ByteBuffer readBuffer() {
 		try {
 			response = CANJNI.FRCNetCommCANSessionMuxReceiveMessage(idBuffer, 0x1fffffff, timestamp);
 		}
-		catch (CANMessageNotFoundException e) {}
+		catch (CANMessageNotFoundException e) {
+			LogKitten.w(e.getMessage()); // We are not actually sure when this error is used (no docs), so just warn for now
```

Comment:
Are you sure `LogKitten.e` isn't appropriate? If we don't know what the error is, all the more reason to err on the side of caution (we can't guarantee it's safe).